eodag/resources/providers.yml

```diff
@@ -2482,48 +2460,29 @@
           downloadLink: $.json.asset.value.href
     products:
       AG_ERA5:
-        output_extension: .zip
         extract: true
       ERA5_SL:
-        output_extension: .grib
       ERA5_PL:
-        output_extension: .grib
       ERA5_PL_MONTHLY:
-        output_extension: .grib
       ERA5_LAND:
-        output_extension: .grib
       ERA5_LAND_MONTHLY:
-        output_extension: .grib
       ERA5_SL_MONTHLY:
-        output_extension: .grib
       UERRA_EUROPE_SL:
-        output_extension: .grib
       GLACIERS_DIST_RANDOLPH:
-        output_extension: .zip
         extract: true
       SATELLITE_CARBON_DIOXIDE:
-        output_extension: .zip
         extract: true
       SATELLITE_FIRE_BURNED_AREA:
-        output_extension: .zip
         extract: true
       SATELLITE_METHANE:
-        output_extension: .zip
         extract: true
```

Comment:
can we have this automatically defined ? if product is zip then we extract ? at least for this provider ?

Reply:
> can we have this automatically defined ? if product is zip then we extract ? at least for this provider ?

yes, `extract: true` can be set in download plugin root settings

Reply:
Does this require a change in the PR or should it be addressed in a different PR?

Reply:
Response: yes this needs to be fixed as part of this PR

Reply:
I don't know how to do this. Had a looked at `eodag/plugins/download/base.py` to no avail.

Reply:
For this particular product and its sibling, the corresponding HTTPDownload plugin configuration has `extract: false`. My understanding is that this sets the default for all products of this provider and it is then overriden to `extract: true` by some products. So unless I'm mistaking, just removing the product-specific `extract: true` will cause the behavior to change. This is the only place in `providers.yml` where I have found the `extract` set on a per-product basis.

Reply:
In fact, having done a bit more testing, it turns out that the `extract: true` at the product has no effect. Since `extract: true` is the default at the download plugin level, I've removed it from both places.